src/test/java/uk/gov/hmcts/sptribs/citizen/event/CicSubmitCaseEventTest.java

```diff
@@ -187,16 +185,15 @@ void shouldUpdateCaseDetails() {
         updatedCaseDetails.setData(caseData);
         when(request.getHeader(AUTHORIZATION)).thenReturn(TEST_AUTHORIZATION_TOKEN);
         when(idamService.retrieveUser(TEST_AUTHORIZATION_TOKEN)).thenReturn(TestDataHelper.getUser());
-        //When
-        AboutToStartOrSubmitResponse<CaseData, State> response1 =
+
+        final AboutToStartOrSubmitResponse<CaseData, State> response =
             cicSubmitCaseEvent.aboutToSubmit(updatedCaseDetails, beforeDetails);
 
-        //Then
-        assertThat(response1).isNotNull();
-        assertThat(response1.getData().getDssCaseData().getOtherInfoDocuments()).isEmpty();
-        assertThat(response1.getData().getCicCase().getApplicantDocumentsUploaded()).hasSize(2);
-        assertThat(response1.getData().getCicCase().getRepresentativeContactDetailsPreference()).isEqualTo(ContactPreferenceType.EMAIL);
-        assertThat(response1.getData().getCicCase().getApplicantDocumentsUploaded().get(0).getValue().getDocumentCategory())
+        assertThat(response).isNotNull();
+        assertThat(response.getData().getDssCaseData().getOtherInfoDocuments()).isEmpty();
+        assertThat(response.getData().getCicCase().getApplicantDocumentsUploaded()).hasSize(2);
+        assertThat(response.getData().getCicCase().getRepresentativeContactDetailsPreference()).isEqualTo(ContactPreferenceType.EMAIL);
+        assertThat(response.getData().getCicCase().getApplicantDocumentsUploaded().get(0).getValue().getDocumentCategory())
             .isIn(DocumentType.DSS_OTHER, DocumentType.DSS_SUPPORTING, DocumentType.DSS_TRIBUNAL_FORM);
```

Comment:
One test case can cover empty/null values for RepresentativeFullName, SupportingDocuments, TribunalFormDocuments and OtherInfoDocuments.
And another test case can cover with values for all.

Reply:
Created https://tools.hmcts.net/jira/browse/DTSSTCI-366 to resolve this.